tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -2211,7 +2254,8 @@ TEST (tensor_stream_test, audio_u16_1000f)
 
   gst_element_set_state (g_test_data.pipeline, GST_STATE_PLAYING);
   g_main_loop_run (g_test_data.loop);
-  g_usleep (jitter);
+
+  EXPECT_TRUE (_wait_pipeline_process_buffers (num_buffers / 2));
```

Comment:
is there any reason checking only 250 frames?

Reply:
The number of frames per buffer of the tensor converter is twice the number of fames of the source.
```
    case TEST_TYPE_AUDIO_U16_1000F:
      /** audio sample rate 16000 (16 bits, unsigned, little endian), 1000 frames */
      str_pipeline =
          g_strdup_printf
          ("audiotestsrc num-buffers=%d samplesperbuffer=500 ! audioconvert ! audio/x-raw,format=U16LE,rate=16000 ! "
          "tensor_converter frames-per-tensor=1000 ! tensor_sink name=test_sink",
          option.num_buffers);
```